Core/Resource/ResourceManager.cs

```diff
@@ -466,7 +466,7 @@ public uint CreateVertexShaderFromFile(string srcFile, string entryPoint, string
 
         public bool CreatePixelShaderFromSource(string shaderSource, string name, string entryPoint, ref uint resourceId)
         {
-            if (string.IsNullOrEmpty(shaderSource) || string.IsNullOrEmpty(entryPoint))
+            if (string.IsNullOrWhiteSpace(shaderSource) || string.IsNullOrWhiteSpace(entryPoint))
```

Comment:
I ran into this while testing - if you have a [CustomPixelShader] and backspace everything and just press Space, T3 crashes. Now it doesn't :)